src/ol/disposable.js

```diff
@@ -0,0 +1,30 @@
+goog.provide('ol.Disposable');
+
+/**
+ * Objects that need to clean up after themselves.
+ * @constructor
+ */
+ol.Disposable = function() {};
```

Comment:
Struct is needed for https://github.com/openlayers/ol3/pull/4793 ; I can make the change after your PR is merged and I rebase on top of it.